ext/mixed/js/api.js

```diff
@@ -171,3 +179,11 @@ function _apiInvoke(action, params={}) {
 function _apiCheckLastError() {
     // NOP
 }
+
+yomichan.on('log', async ({error, level, context}) => {
+    try {
+        await apiLog(errorToJson(error), level, context);
+    } catch (e) {
+        // NOP
+    }
+});
```

Comment:
This will save some lines if it's here, but just wondering if it would be more clear if this was moved to the corresponding `*-main.js` files.

```
content-script-main.js
float-main.js
popup-preview-frame-main.js
search-main.js
settings/main.js
context-main.js (not used there though)
```

Reply:
I could do that. I initially didn't just because it was less code to put it only here, and the main scripts weren't reorganized at the time.

Reply:
1aa495bfbea02d2656dea8a99019faacdd314e8d